flight/PiOS/STM32F4xx/pios_flash_internal.c

```diff
@@ -34,6 +36,9 @@
 #include "pios_semaphore.h"
 #include <stdbool.h>
 
+/* Todo: consider relocating to a good header */
+bool __attribute__((weak)) vehicle_is_armed = false;
```

Comment:
Yep. Not sure the doxygen thing will pick up Todo (vs. TODO) though. :stuck_out_tongue: